pyvista/plotting/plotting.py

```diff
@@ -3494,8 +3494,8 @@ def orbit():
                 if write_frames:
                     self.write_frame()
 
-
-        if bkg and isinstance(self, pyvista.BackgroundPlotter):
+        is_background_plotter = 'BackgroundPlotter' in str(type(self))
+        if bkg and is_background_plotter:
```

Comment:
This isn't okay. The code here is assuming a feature for a downstream package. The `BackgroundPlotter` downstream should override this method with this background threading feature

Reply:
We should also double-check that we don't do anything similar in other places. A simple grep for `BackgroundPlotter` showed nothing else but something in the back of my mind says there may be another splot in `BasePlotter` (or its inherited classes) where we check if it is a `BackgroundPlotter`

Reply:
Agreed.  I'll patch this.